Makefile

```diff
@@ -79,67 +79,14 @@ prepare-release: cross
 test:
 	go test -race $(PKGS)
 
-# Run main e2e tests
-.PHONY: test-main-e2e
-test-main-e2e:
-ifdef TIMEOUT
-	go test -v github.com/redhat-developer/odo/tests/e2e --ginkgo.focus="odoe2e" -ginkgo.succinct -timeout $(TIMEOUT)
-else
-	go test -v github.com/redhat-developer/odo/tests/e2e --ginkgo.focus="odoe2e" -ginkgo.succinct
-endif
-
-# Run component e2e tests
-.PHONY: test-cmp-e2e
-test-cmp-e2e:
-ifdef TIMEOUT
-	go test -v github.com/redhat-developer/odo/tests/e2e --ginkgo.focus="odoCmpE2e" -ginkgo.succinct -timeout $(TIMEOUT)
-else
-	go test -v github.com/redhat-developer/odo/tests/e2e --ginkgo.focus="odoCmpE2e" -ginkgo.succinct -timeout 15m
-endif
-
-# Run java e2e tests
-.PHONY: test-java-e2e
-test-java-e2e:
-ifdef TIMEOUT
-	go test -v github.com/redhat-developer/odo/tests/e2e --ginkgo.focus="odoJavaE2e" -ginkgo.succinct -timeout $(TIMEOUT)
-else
-	go test -v github.com/redhat-developer/odo/tests/e2e --ginkgo.focus="odoJavaE2e" -ginkgo.succinct
-endif
-
-# Run source e2e tests
-.PHONY: test-source-e2e
-test-source-e2e:
-ifdef TIMEOUT
-	go test -v github.com/redhat-developer/odo/tests/e2e --ginkgo.focus="odoSourceE2e" -ginkgo.succinct -timeout $(TIMEOUT)
-else
-	go test -v github.com/redhat-developer/odo/tests/e2e --ginkgo.focus="odoSourceE2e" -ginkgo.succinct
-endif
-
-# Run service catalog e2e tests
-.PHONY: test-service-e2e
-test-service-e2e:
-ifdef TIMEOUT
-	go test -v github.com/redhat-developer/odo/tests/e2e --ginkgo.focus="odoServiceE2e" -ginkgo.succinct -timeout $(TIMEOUT)
-else
-	go test -v github.com/redhat-developer/odo/tests/e2e --ginkgo.focus="odoServiceE2e" -ginkgo.succinct
-endif
-
-# Run link e2e tests
-.PHONY: test-link-e2e
-test-link-e2e:
-ifdef TIMEOUT
-	go test -v github.com/redhat-developer/odo/tests/e2e --ginkgo.focus="odoLinkE2e" -ginkgo.succinct -timeout $(TIMEOUT)
-else
-	go test -v github.com/redhat-developer/odo/tests/e2e --ginkgo.focus="odoLinkE2e" -ginkgo.succinct
-endif
-
 # Run all e2e tests
 .PHONY: test-e2e
+test-e2e: ODO_TEST_SCENARIO=""
 test-e2e:
```

Comment:
I am by no means a `Makefile` expert so I have to ask, is this correct?
I mean does it make sense to have 

```
test-e2e: ODO_TEST_SCENARIO=""
test-e2e:
```

?

Reply:
Me neither ;)
Syntax seems to be correct as it does not throw any make error. I can check if this can be done in a better way